lib/content/utilities.ex

```diff
@@ -54,6 +54,7 @@ defmodule Content.Utilities do
 
   def destination_for_prediction(_, 1, "70209"), do: {:ok, :lechmere}
   def destination_for_prediction(_, 1, "70205"), do: {:ok, :north_station}
+  def destination_for_prediction(_, 1, "70503"), do: {:ok, :union_square}
   def destination_for_prediction(_, 1, "70201"), do: {:ok, :government_center}
```

Comment:
The spec for this function isn't explicit.  Parameters are:  route_id, digit to represent directionality of the terminus, stop_id?

Reply:
Yep basically. I guess it'd be more accurate to say `Line name`, or whatever the more technical term is. I _think_ a route Id isn't necessarily synonymous with the name of the line, but I may be wrong. Yep, the bit represents the directionality, and the stop_id is used for specific cases where a stop is a commonly used terminus. If the predictions are not for a terminus defined in these specific cases, it'll default to using the terminus for the line which are defined in lines 66-73 for the Green line, for example

Reply:
The first parameter is actually the route ID here. "Line" is an "experimental" GTFS extension that [the MBTA uses](https://github.com/mbta/gtfs-documentation/blob/master/reference/gtfs.md#linestxt), and would be [`line-Green`](https://api-v3.mbta.com/lines/line-Green), referring to the set of routes, `Green-B`, `Green-C`, `Green-D`, and `Green-E`.

This function actually does match against route as the first argument. (You'll see some clauses that match against `Green-B`, for example. Others match against `Red`, but confusingly `Red` is a route (with `line-Red` as the line ID).

In this set of functions we omit the route altogether so that, e.g., `"70205"` for _any_ route will match, but really we mean any Green line route (and no other route will have that stop ID). For clarity, the pattern could be `"Green-" <> _`, I suppose.